build.js

```diff
@@ -179,7 +168,14 @@ async function build() {
 
   const swBundle = await rollup.rollup({
     input: 'src/lib/sw.js',
-    external: disallowExternal,
+    manualChunks: (id) => {
+      const chunkNames = ['idb-keyval', 'virtual', 'workbox'];
```

Comment:
This is probably fine but I always feel awkward matching paths against this.

What about:

```js
if (!path.isAbsolute(id)) {
  return id.startsWith('virtual:');
}
const chunks = ['node_modules/idb-keyval/', 'node_modules/workbox/'];
const rel = path.relative(__dirname, id);
return chunks.includes(rel);
```

I'm happy to be disagreed with.

Reply:
Yeah, feel free to change this logic based on your desired chunk granularity.

Except that [`manualChunks`](https://rollupjs.org/guide/en/#manualchunks) expects you to return either `undefined` (keep things in the top-level output) or a `string` (move the code to the chunk with that name), and in your sample code, you're returning `boolean`.

Given that, what would you like this code to look like?